Makefile

```diff
@@ -4,6 +4,8 @@ PKG?=./pkg/... ./internal/...
 
 GO_MODULE ?= $(shell go list -m)
 GO_FILES  = $(shell find ./ -name '*.go' -not -name '*_test.go')
+GO_FILES  += ./go.mod
+GO_FILES  += ./go.sum
```

Comment:
Dumb question maybe, but cannot be written like this?
```
GO_FILES+=./go.mod ./go.sum
```

Seems to work on my lab, but maybe you prefer to keep it in two lines for better readability.

Reply:
Yep, it was only a matter of my own personal taste :+1: